styles/main.scss

```diff
@@ -649,3 +649,30 @@ span {
     font-size: 26px;
   }
 }
+/* --------------------------- */
+/* ----- countdown block ----- */
+/* --------------------------- */
+
+.countdown-block {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #F5F5F5;
+  padding: 12px 10px;
+  gap: 8px;
+
+  span {
+    font-weight: 400 !important;
+    font-size: 15px !important;
+  }
+
+  .num {
+    font-weight: 500 !important;
+    font-size: 18px !important;
+    line-height: 13px !important;
+  }
+
+  .countdown-icon {
+    font-size: 18px !important;
+  }
```

Comment:
please can you avoid using to much `important` this can overrides all other declarations and makes the CSS code more difficult to maintain and debug

Reply:
youcan scope the countdown ccs styles in the countdown.css, and change the original countdown styles
do we have the same countdown style everywhere ?

Reply:
we want to avoid using bad practices

Reply:
adil has changed the styling of the countdown and I didn't want to touch that CSS file instead I have take a CSS class and put it in the main with !important to override the css of the countdown

Reply:
I have did it but the font size didn't want to change that like whose the first file was readed in the dom